lib/galaxy/model/migrate/versions/0169_add_missing_indexes.py

```diff
@@ -15,100 +15,130 @@
 metadata = MetaData()
 
 indexes = [
+    [
```

Comment:
You shouldn't touch the 0169 migrations. The 0169 should be merged forward, any additional changes should go to 0170.

Reply:
Oh, OK (I thought we were adding all of these to 20.09 as one migration) Fixing.